### PR TITLE
Reworded "list of objects" to remove any ambiguity

### LINKED ISSUE #30727: In the context of arguments, what exactly is "a list of <x>"?
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/console

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/API/console#outputting_text_to_the_console

### What information was incorrect, unhelpful, or incomplete?

This is a question with the potential for a follow-up PR. It's about this paragraph, emphasis mine:

> There are two ways to use each of the output methods; you can pass in **a list of objects** whose string representations get concatenated into one string, then output to the console, or you can pass in a string containing zero or more substitution strings followed by a list of objects to replace them.

This might be overly pedantic, but my first interpretation of this phrase was `console.log([obj1, ... objN])`. I'm 99% sure it meant `console.log(obj1, ... objN)` instead (no array). I considered creating a PR to remove "the ambiguity," but I think I need clarity first, as this could just be my misunderstanding. 

1) Is this a situation where the English language is imprecise and it could mean both?
2) Was my initial interpretation incorrect?
3) Or does `a list of <x>` precisely mean passing `x1, ... xN` and not `[x1, ... xN]`?

Depending on the answer(s), I might attempt to clarify details in this paragraph and a couple of following sentences.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/console`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/console
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/console/index.md
* Last commit: https://github.com/mdn/content/commit/c367939020b2cbd60da7bd56a14670659d5e9491
* Document last modified: 2023-11-29T07:09:38.000Z

</details>

### PR DESCRIPTION
**Pull request by DanKaplanSES**
### Description, Motivation, and related issues

Fixes #30727

### CODE REVIEW

**Comment by teoli2003** [comment]
@DanKaplanSES A technicality: I've added the word _Fixes_ in your description: that way when this PR is merged, GitHub will automatically close the issue (that I reopen).

That's the standard way of doing it.

**Comment by DanKaplanSES** [comment]
> @DanKaplanSES A technicality: I've added the word _Fixes_ in your description: that way when this PR is merged, GitHub will automatically close the issue (that I reopen).
> 
> That's the standard way of doing it.

Oh! Thank you. I had know idea.

File: files/en-us/web/api/console/index.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ This page documents the [Methods](#methods) available on the `console` object an
...
 
-There are two ways to use each of the output methods; you can pass in a list of objects whose string representations get concatenated into one string, then output to the console, or you can pass in a string containing zero or more substitution strings followed by a list of objects to replace them.
+There are two ways to use each of the output methods; you can pass in a variable number of arguments whose string representations get concatenated into one string, then output to the console, or you can pass in a string containing zero or more substitution strings followed by a variable number of arguments to replace them.
```
Comment (wbamberg) [inline files/en-us/web/api/console/index.md:79]: Do you think it is worth bulleting this to make it easier to read?

e.g.

```suggestion
There are two ways to use each of the output methods:
- Pass in a variable number of arguments whose string representations get concatenated into one string, then output to the console.
- Pass in a string containing zero or more substitution strings followed by a variable number of arguments to replace them.
```

just a suggestion, if you think it is worth doing.

Comment (DanKaplanSES) [reply files/en-us/web/api/console/index.md:79]: Yep, I considered that. I think that's a good idea.

**Review by wbamberg — APPROVED** [review]
👍 thank you @DanKaplanSES !

**Pull request merged**